### PR TITLE
More misc. typos

### PR DESCRIPTION
**Pull request by luzpaz**
Follow-up to #10261

### CODE REVIEW

**Comment by tacaswell** [comment]
Thanks!

Do you think it would be possible to run codespell on travis to make sure we do not put spelling errors in to begin with?

File: lib/matplotlib/backend_tools.py [hunk 1]
```diff
@@ -228,7 +228,7 @@ def set_figure(self, figure):
             if figure:
                 self.trigger(self, None)
             else:
-                # if there is no figure, triggen wont change the internal state
+                # if there is no figure, trigger wont change the internal state
```
Comment (QuLogic) [inline lib/matplotlib/backend_tools.py]: should be `won't` as well.

File: lib/matplotlib/dates.py [hunk 2]
```diff
@@ -1422,15 +1422,15 @@ def autoscale(self):
 
 class MonthLocator(RRuleLocator):
     """
-    Make ticks on occurances of each month month, e.g., 1, 3, 12.
+    Make ticks on occurrences of each month month, e.g., 1, 3, 12.
```
Comment (QuLogic) [inline lib/matplotlib/dates.py]: Double 'month', I think.

File: lib/matplotlib/patches.py [hunk 3]
```diff
@@ -2831,7 +2831,7 @@ def __reduce__(self):
     class Arc3(_Base):
         """
         Creates a simple quadratic bezier curve between two
-        points. The curve is created so that the middle contol points
+        points. The curve is created so that the middle control points
```
Comment (QuLogic) [inline lib/matplotlib/patches.py]: `middle control point` (not plural)

File: lib/matplotlib/offsetbox.py [hunk 4]
```diff
@@ -626,7 +626,7 @@ def set_offset(self, xy):
         """
         set offset of the container.
 
-        Accept : tuple of x,y cooridnate in disokay units.
+        Accept : tuple of x,y coordinate in disokay units.
```
Comment (QuLogic) [inline lib/matplotlib/offsetbox.py]: s/disokay/display/

File: lib/matplotlib/tri/trirefine.py [hunk 5]
```diff
@@ -285,7 +285,7 @@ def _refine_triangulation_once(triangulation, ancestors=None):
 
         # Now dealing with slave elems.
         # for each slave element we identify the master and then the inode
-        # onces slave_masters is indentified, slave_masters_apex is such that:
+        # onces slave_masters is identified, slave_masters_apex is such that:
```
Comment (QuLogic) [inline lib/matplotlib/tri/trirefine.py]: once

**Comment by luzpaz** [comment]
Made changes. Please let me know when and I'll squash the commits. 

> Do you think it would be possible to run codespell on travis to make sure we do not put spelling errors in to begin with?  

@tacaswell Yes, check out https://github.com/lucasdemarchi/codespell/issues/139 for some methodologies.

**Comment by jklymak** [comment]
This has a couple of PEP8 lines that are too long, and needs a rebase...

**Comment by luzpaz** [comment]
@jklymak shall I use `flake8` to check said issues? I checked `flake8` and there is a lot of output. Not sure where to start.

**Comment by jklymak** [comment]
Just drill down on the Travis-CI python 3.6 test. It lists the pep8 errors that we care about...  You have a couple of lines that are too long...

**Comment by luzpaz** [comment]
@jklymak pep8 fixes submitted

**Pull request merged**